includes/class-admin-settings.php

```diff
@@ -539,18 +569,18 @@ class="regular-text"
 	public function sanitize_settings( $input ) {
 		$sanitized_input = array();
 
-		$sanitized_input['enable']         = ( isset( $input['enable'] ) && $input['enable'] ) ? 1 : 0;
+		$sanitized_input['enable']         = ( ! empty( $input['enable'] ) && $input['enable'] ) ? 1 : 0;
```

Comment:
Why the change here for `enable` and later for `enable_debug`, and is this needed for multisite support?

Reply:
Also, the second condition shouldn't be necessary here if using `! empty()`

Reply:
These values are unset by default. With multisite the sanitize function runs twice, I assume once from the `register_setting()` callback. The second time this runs the input is `0` and therefore it is set, but it is also empty.

Reply:
Let me test that.

Reply:
Tested and working.

Reply:
Further simplified, doesn't need ternary.